Fenrir/Commands.fs

```diff
@@ -12,7 +12,14 @@ open Fenrir.Tools
 open Fenrir.Zlib
 
 let getRawObjectPath (gitDirectoryPath: string) (objectHash: string): string =
-    Path.Combine(gitDirectoryPath, "objects", objectHash.Substring(0, 2), objectHash.Substring(2, 38))
+    if objectHash.StartsWith("ref: ")
```

Comment:
To be honest, I don't like this solution very much.

For a function named `getRawObjectPath (gitDirectoryPath: string) (objectHash: string)`, I would expect that it wants a _hash_, and not will accept a string like `ref: some/ref` instead of a hash.

Current solution introduces an undocumented "feature" where I can use a ref-like string almost everywhere instead of an object hash.

Could we resolve this `ref:` in the function that reads the hash from the ref file instead?

Reply:
In the latest commit that logic moved in the `readRefsRecursively` function at Fenrir/Refs.fs file.